package.json

```diff
@@ -56,9 +56,14 @@
         "prettier": "prettier --write '**/*.js'",
         "prettier_check": "prettier --check '**/*.js'",
         "prettier_debug": "prettier --debug-check '**/*.js'",
+        "eslint-check": "eslint '**/*.js'",
+        "eslint-pre-push": "scripts/eslint-pre-push.sh",
```

Comment:
could you move these up so they're under the `eslint` item?

Reply:
I've alphabetized the list, which reordered other entires as well.